extensions/objects/templates/normalized-string-editor.component.ts

```diff
@@ -22,18 +22,27 @@ import { Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, S
 import { downgradeComponent } from '@angular/upgrade/static';
 import { Subscription } from 'rxjs';
 import { ExternalSaveService } from 'services/external-save.service';
+
+type NormalizedStringEditorComponentArgs = {
+  currentValue?: { largeInput: string },
+  previousValue?: { largeInput: string }
```

Comment:
Why are these optional ?

Reply:
referring to [this](https://github.com/oppia/oppia/pull/13598/files#diff-ad16fe0512aa3873b573c43b2ff0eed2e8066175d99fe7e137881a98fcf5a128R81-R86). It is possible that in the form the current and previous values have not been specified.

Reply:
Should comment be added?

Reply:
added comment